src/app/containers/PageHandlers/withData.jsx

```diff
@@ -1,36 +1,55 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { shape, element } from 'prop-types';
 import articlePropTypes from '../../models/propTypes/article';
 import ErrorMain from '../ErrorMain';
-import nodeLogger from '../../lib/logger.node';
 import deepGet from '../../lib/utilities/deepGet';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import getPassportHome from '../../lib/utilities/getPassportHome';
 
-const logger = nodeLogger(__filename);
-
-// checks for data and status keys, setting default status if not found
+// checks for data, status, setting default status if not found
 const constructRenderObject = data => ({
   status: deepGet(['status'], data) || 500,
   pageData: deepGet(['pageData'], data),
 });
 
-// checks for pageData and 200 status
+const validatePassportHome = passportHome => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Why are we ignoring this error?

Reply:
Thanks @rhell, I needed to get the `service` from context but it seems it's been passed down as props already